src/components/slider/slider.tsx

```diff
@@ -0,0 +1,433 @@
+import { useTheme } from 'native-base';
+import React, { useState, useRef, useEffect, useMemo } from 'react';
+import { View, Text, PanResponder, Animated, LayoutChangeEvent } from 'react-native';
+
+import { SliderProps } from '../../types/slider';
+import { Input } from '../inputs';
+
+import { useSliderStyles } from './slider.styles';
+
+enum BubbleHandle {
```

Comment:
From a slider component point of view, `BubbleHandle` is not immediately clear what it represents.